css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third {
+  width: 100%;
```

Comment:
remove all of the widths set here, you don't want to set both width and flex basis